examples/vic-gov-au/nuxt.config.js

```diff
@@ -118,7 +118,7 @@ export default {
   * See https://www.npmjs.com/package/@dpc-sdp/ripple-nuxt-tide
   */
   tide: {
-    baseUrl: process.env.CONTENT_API_SERVER,
+    baseUrl: 'http://nginx-php-content-vic-pr-999.lagoon.vicsdp.amazee.io/',
```

Comment:
```suggestion
    baseUrl: process.env.CONTENT_API_SERVER,
```
Revert when QA is done